README.md

```diff
@@ -278,11 +278,11 @@ Variables in vars.yml
    -  ** `type: basics`**
       - `kernel_message`: load `imklog`. Default to `false`.
       - `use_imuxsock`: use `imuxsock` instead of `imjournal`. Default to `false`.
-      - `rsyslog_imjournal_ratelimit_burst`: set to imjournal RateLimit.Burst. Default to 20000.
-      - `rsyslog_imjournal_ratelimit_interval`: set to imjournal RateLimit.Interval. Default to 600.
-      - `rsyslog_imjournal_persist_state_interval`: set to imjournal PersistStateInterval. Default to 10.
+      - `journal_ratelimit_burst`: set to imjournal RateLimit.Burst. Default to 20000.
+      - `journal_ratelimit_interval`: set to imjournal RateLimit.Interval. Default to 600.
+      - `journal_persist_state_interval`: set to imjournal PersistStateInterval. Default to 10.
    -  ** `type: files`**
-      - `rsyslog_input_log_path`: File name to be read by the imfile plugin. The value should be full path. Wildcard '*' is allowed in the path.  Default to `/var/log/containers/*.log`.
+      - `input_log_path`: File name to be read by the imfile plugin. The value should be full path. Wildcard '*' is allowed in the path.  Default to `/var/log/containers/*.log`.
```

Comment:
Is the default still `/var/log/containers/*.log`?

Reply:
Yeah, I haven't changed it. Do you have any recommendation?

Reply:
> Yeah, I haven't changed it. Do you have any recommendation?

I don't think there is a default.  I think this is a required parameter?  It doesn't make sense for the user to specify a file input with no file to read from . . .

Reply:
> > Yeah, I haven't changed it. Do you have any recommendation?
> 
> I don't think there is a default. I think this is a required parameter? It doesn't make sense for the user to specify a file input with no file to read from . . .

Done in https://github.com/linux-system-roles/logging/pull/131/commits/9b8ebe69a1fcc24274cd1138bb44e5bde5e5c40e. (I think this commit also should not be squashed, but be a separate commit.)